src/internal/menu/tabs/SessionsTab.tsx

```diff
@@ -1,59 +1,39 @@
-import React from 'react';
-import {Container, Card, Col, Nav, Row, Tab, Button} from 'react-bootstrap';
+import React, {useMemo} from 'react';
+import {Container, Col, Nav, Row, Button} from 'react-bootstrap';
 import {Link, Element} from 'react-scroll';
-import {UniqueIdManager} from 'phovea_core';
-import {CurrentSessionCard, SavedSessionCard, TemporarySessionCard, UploadSessionCard} from '../../components';
-
-
-
-
+import {PluginRegistry, UniqueIdManager} from 'phovea_core';
+import {useAsync} from '../../../hooks';
+import {EXTENSION_POINT_START_MENU, IStartMenuSectionDesc} from '../../..';
 
+function byPriority(a: any, b: any) {
+  return (a.priority || 10) - (b.priority || 10);
+}
 
 export function SessionsTab() {
-
-  // Tody merge the cards CurrentSessionCard, SavedSessionCard, TemporarySessionCard
-  const cards = [
-    {
-      id: 'current',
-      headerText: 'Current Session',
-      getElement: () => <CurrentSessionCard />
-    },
-    {
-      id: 'saved',
-      headerText: 'Saved Session',
-      getElement: () => <SavedSessionCard />
-    },
-    {
-      id: 'temporary',
-      headerText: 'Temporary Session',
-      getElement: () => <TemporarySessionCard />
-    },
-    {
-      id: 'import',
-      headerText: 'Import Session',
-      getElement: () => <UploadSessionCard />
-    }
-  ];
-
-
   const suffix = UniqueIdManager.getInstance().uniqueId();
+  const loadSections = useMemo(() => () => {
+    const sectionEntries = PluginRegistry.getInstance().listPlugins(EXTENSION_POINT_START_MENU).map((d) => d as IStartMenuSectionDesc).sort(byPriority);
```

Comment:
We cannot reuse the general `EXTENSION_POINT_START_MENU` extension point here. Previously, this extension point was meant to add new sections to the start menu. Now you are repurposing it to add session cards only. I'd suggest to remove the `EXTENSION_POINT_START_MENU` and create a new one for the sessions tab (like you did for the dataset tab). Please take a note the removal of the EP as breaking change in the PR and add the major label.